politeiawww/cmd/pictl/cmdproposalbillingstatuschanges.go

```diff
@@ -45,20 +45,30 @@ func (c *cmdProposalBillingStatusChanges) Execute(args []string) error {
 		return err
 	}
 
-	// Print billing status changes
-	for _, bsc := range bscsr.BillingStatusChanges {
-		printBillingStatusChange(bsc)
-		printf("-----\n")
+	// Print billing status changes for all tokens
+	for t, bscs := range bscsr.BillingStatusChanges {
+		printf("Proposal %v\n", t)
+		if len(bscs) > 0 {
+			for i, bsc := range bscs {
+				printBillingStatusChange(bsc)
+				if i != len(bscs)-1 {
+					printf("-----\n")
```

Comment:
```suggestion
					printf("  -----\n")
```

So that is aligns with the billing status summaries.

```
Proposal 1c10c65d42b50607
  Token    : 1c10c65d42b50607
  Status   : completed
  PublicKey: 0ddecc1f086679847211f5cead033bc483ab0b53b834222987ee68a2f33cb5e8
  Signature: a23de69a5af826f07b6c8e4d8ef6772ef0c04f699b08f03646a259b1b05fdf9f18fd5b4b344325319b760e823b89912daff71eb571f4485e7c1d8b2461a53e0b
  Receipt  : 6cc649565115ca0f4b77c8f19292ed13092cc996dd10b731ea0423347016b511bb21ac2783b0919749bf687790ce966175f37aff8934aa2ca04133c8f0d9e20c
  Timestamp: 10/10/2021 9:32am
  -----
  Token    : 1c10c65d42b50607
  Status   : closed
  Reason   : Did not deliver work.
  PublicKey: 0ddecc1f086679847211f5cead033bc483ab0b53b834222987ee68a2f33cb5e8
  Signature: e74e0b4da6e8dcda66243432ea992741ae40a5c66f702068107183d3abd9a31e14da093abedf0fbe6087a2ab0ed8209153036e3ddb9bfc2386baafbd3fc62902
  Receipt  : 780d8233d84d682139fa21746750438e4997fc43c7217b263e0cd5271699dfde41c45b67a41f322ea0f1ce44d87e22cd16fc7f2d18227a595a8c316c7aaf9501
  Timestamp: 10/10/2021 9:33am

```